concourse/steps/notification.mako

```diff
@@ -249,4 +250,22 @@ mailutil.notify(
     email_cfg_name=email_cfg_name,
     recipients=email_cfg['recipients'],
 )
+% if on_error_cfg.slack():
+slack_cfg = ${on_error_cfg.slack()}
+cfg_factory = ci.util.ctx().cfg_factory()
+for channel_cfg in slack_cfg.get('channel_cfgs', []):
+  slack_cfg_name = channel_cfg['slack_cfg_name']
+  slack_helper = slackclient.util.SlackHelper(cfg_factory.slack(slack_cfg_name))
+  for slack_channel in channel_cfg['channel_name']:
+    logger.info(channel_cfg['channel_name'])
+    try:
+      slack_helper.post_to_slack(
+        channel=slack_channel,
+        title=email_cfg['subject'],
+        message=body
+      )
+    except:
```

Comment:
Is there a specific error you want to catch, e.g. because it may happen sporadically? Then please specify the concrete error instead of catching all exceptions. Otherwise, I'd suggest to remove the error handling because it rather results in silently ignoring errors (nobody is likely to look into the notification logs in case no error is reported -> hence the problem might not be realised for quite some time).